packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -302,7 +325,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
       }
     }
 
-    decryptAll();
+    decryptAll().catch(err => console.log(err));
```

Comment:
Was this used for debugging or are you intending to keep this here? (No issues either way, just wanted to make sure it was intended)

Reply:
this was causing the app to crash under certain conditions during my testing. I don't remember the exact situation, but it for whatever reason the decryption was failing and throwing an error.  Typically i would call a dispatch to present the error to user, but I didnt see a way to do that, so just logged. Can silently just catch this, but definitely didn't want app to crash.